pages/store/claim.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <main class="flex flex-col justify-center items-center min-h-svh mb-safe py-[56px]">
+    <template v-if="!hasLoggedIn">
+      <p
+        class="text-center text-gray-500 mb-4"
+        v-text="$t('claim_page_login_cta_description')"
+      />
+      <LoginPanel
+        class="max-w-[348px] w-full p-6 pt-8 rounded-[20px] bg-white"
+        @connect="handleLogin"
+      />
+    </template>
+    <BookLoadingScreen
+      v-else
+      :book-name="bookInfo?.name.value"
+      :book-cover-src="bookCoverSrc"
+      icon="i-material-symbols-check-circle-rounded"
+      :icon-label="$t('claim_page_purchase_successful_label')"
+      :loading-label="$t('claim_page_loading_label')"
+    >
+      <template
+        v-if="hasStartedClaimFlow && isClaimed && !isClaiming && !isCheckingItemsDelivery"
```

Comment:
Can we simplify this? e.g. If `isClaimed` is set, `hasStartedClaimFlow` must be true and `!isClaiming` must have passed